source/_components/group.markdown

```diff
@@ -42,24 +43,32 @@ group:
 
 Configuration variables:
 
+- **view** (*Optional*): If yes then the entry will be shown as a view (tab).
 - **name** (*Optional*): Name of the group.
-- **icon** (*Optional*): An optional icon to show in the Frontend.
-- **view** (*Optional*): If yes then the entry will be shown as a view.
-- **entities** array or comma delimited string (*Required*): List of entites to group.
+- **icon** (*Optional*): If the group is a view, this icon will show at the top in the frontend instead of the name. If it's not a view, then the icon shows when this group is used in another group.
+- **entities** (*Required*): array or comma delimited string, list of entities to group.
 
 <p class='img'>
 <img src='/images/blog/2016-01-release-12/views.png'>
 Example of groups shown as views in the frontend.
 </p>
 
-If all entities are switches or lights they can be controlled as one with a switch at the top of the card. Grouped states should share the same type of states (ON/OFF or HOME/NOT_HOME).
+If all entities in a group are switches or lights then Home Assistant adds a switch at the top of the card that turns them all on/off at once.
```

Comment:
I have groups with a mix of devices and Home Assistant still adds a switch on top of the card. 
Groups with devices that have an **Assumed State** will have this inherited from these devices and will show the corresponding switch for the group.